docs/hfvqe/Making MolecularData.ipynb

```diff
@@ -0,0 +1,566 @@
RSSxRcQEIk3RiGihJLaIaJ8ktohonSS2iGiV7O4REa2UxBYRbZMlVbNR0fyYqna+PeiRMu+L/n9PPX/A/yuImCRN0Yhol0zQjYhWSmKLiDbJyoOIaCWNDXZmS2KLiIla0MdWalhQ0sck3SHpdkmXS8qLOiNaQC52NNWcE5ukVcBHgGHbJwBDwOlVBRYRNXLBowRJyyVdK+me7tdlU9zzKkk/71agbpP03iJll53ItRh4rqTFwCHAAyXLi4gG6FON7TzgOtvHAdd1ryd7Avgr238ErAMulPS8mQqec2KzfT/wJeA+YBfwqO0fTb5P0kZJI5JGRh9/fK6Pi4h+6kONDVgPbO6ebwbe1ROG/Wvb93TPHwB2Ay+YqeAyTdFl3cCOBV4ELJV05hSBbbI9bHt4aOnSuT4uIvql+5aqIgew4kDFpXtsnMWTVtre1T1/EFj5bDdLOglYAvxmpoLLjIquBX5r++HuQ68EXg9cWqLMiKjZLOex7bE9PG1Z0o+BI6f41vnjL2xbmv6pko4CvgtssD3jGsYyie0+4GRJhwB/AE4FRkqUFxFN4WqGPG2vne57kh6SdJTtXd3EtXua+w4HrgLOt/2LIs8t08d2A7AFuBn4ZbesTXMtLyKao0+DB1uBDd3zDcAPeuKQlgD/CnzH9paiBZcaFbX9WduvsH2C7ffbfqpMeRHRAEUHDsontguAt0q6h07X1gUAkoYlXdy95z3AG4GzJN3aPV41U8FZeRARPfqxH5vtR+h0YU3+fAT4YPf8UubQb5/EFhE9stFkRLSLqWzwoC79TWzqHiUtfryCQsjOtxHTafI60CJSY4uIXklsEdEm2WgyItrHzkaTEdFCg53XktgioleaohHRLgbSFI2I1hnsvJbEFhG90hSNiNbJqGhEtEsLXr+XxBYRE3Qm6A52Zktii4heA76MOoktInqkxhYR7ZI+tohon6wVjYg2SlM0IlrF2Ro8ItooNbZZqOgvwf6l1fzQD95d6u2Dz3jyhQP+5y1issHOa6mxRUQvjQ32H+sktoiYyGSCbkS0i3Am6EZECw14Ypux91zSJZJ2S7p93GfLJV0r6Z7u12XzG2ZE9JVd7GioIsOC3wbWTfrsPOA628cB13WvI6INDvSxFTkaasbEZvtnwN5JH68HNnfPNwPvqjasiKiTxsYKHU011z62lbZ3dc8fBFZWFE9E1K7ZzcwiSg8e2LY0/Q7pkjYCGwGGlqUrLqLxzMAntrlOvX9I0lEA3a+7p7vR9ibbw7aHh5YunePjIqKv2t7HNo2twIbu+QbgB9WEExFNILvQ0VRFpntcDvwceLmknZLOBi4A3irpHmBt9zoi2mLAp3vM2Mdm+4xpvnVqxbFERBPYMNrgdmYBWXkQEb0aXBsrIoktInoNeGKrZkOyiGgPA2MudpQwm6WZkg7v9vH/Y5Gyk9giYhKDx4od5cxmaebngZ8VLbi/TVGBK0ilix9T+ULIzrcRUzL9GjxYD7y5e74ZuB7428k3SXoNndVNVwPDRQpOjS0iehWf7rFC0si4Y+MsnjLj0kxJi4AvA5+cTfgZPIiIXsUHD/bYnrYWJenHwJFTfOv8iY+bdmnmh4BttndKxVtqSWwRMUl1k29tr53ue5IeknSU7V3PsjTzdcAbJH0IOBRYIukx28+6VVoSW0RMZKA/WxIdWJp5AdMszbT9lwfOJZ0FDM+U1CB9bBExlf4sqZpyaaakYUkXlyk4NbaImKQ/S6psP8IUSzNtjwAfnOLzb9PZ0XtGSWwRMZHB5eeo1SqJLSJ6lVxVULcktojoNeBrRZPYImIiu1+jovMmiS0ieqXGFhHtYjw6WncQpSSxRcREB7YtGmBJbBHRK9M9IqJNDDg1tohoFTs1tohon0EfPJD7OKwr6WHgv2e4bQWwpw/hFJV4Zta0mBZyPC+2/YIyBUi6mk7MReyxva7M8+ZDXxNbEZJGnm3jun5LPDNrWkyJJ7JtUUS0ThJbRLROExPbproDmCTxzKxpMSWeBa5xfWwREWU1scYWEVFKEltEtE5jEpukdZLulrRd0oxvoelDPGsk/VTSnZLukHRu3TEBSBqSdIukHzYgludJ2iLpV5LukvS6muP5WPd3dbukyyUdXEMMl0jaLen2cZ8tl3StpHu6X5f1O66FphGJTdIQcBHwNuB44AxJx9cbFfuBT9g+HjgZ+JsGxARwLnBX3UF0fQ242vYrgD+mxrgkrQI+Quf1bCcAQ8DpNYTybWDyhNXzgOtsHwdc172OedSIxAacBGy3fa/tfcAVwPo6A7K9y/bN3fPf0/mfdlWdMUlaDbwDKPVqsopiOQJ4I/BNANv7bP9vrUF1lgg+V9Ji4BDggX4HYPtnwN5JH68HNnfPNwPv6mdMC1FTEtsqYMe4653UnETGk3QMcCJwQ82hXAh8CmjCCuVjgYeBb3WbxhdLWlpXMLbvB74E3AfsAh61/aO64plkpe1d3fMHgZV1BrMQNCWxNZakQ4HvAx+1/bsa43gnsNv2TXXFMMli4NXA122fCDxOjU2sbr/VejoJ90XAUkln1hXPdNyZX5U5VvOsKYntfmDNuOvV3c9qJek5dJLaZbavrDmcU4DTJP0Xnab6WyRdWmM8O4Gdtg/UYrfQSXR1WQv81vbDtp8GrgReX2M84z0k6SiA7tfdNcfTek1JbDcCx0k6VtISOp2+W+sMSJLo9B/dZfsrdcYCYPvTtlfbPobOz+cntmurkdh+ENgh6eXdj04F7qwrHjpN0JMlHdL93Z1KcwZZtgIbuucbgB/UGMuC0Ij92Gzvl3QOcA2d0axLbN9Rc1inAO8Hfinp1u5nf2d7W30hNc6Hgcu6f4zuBT5QVyC2b5C0BbiZzoj2LdSwlEnS5cCbgRWSdgKfBS4A/lnS2XS27XpPv+NaaLKkKiJapylN0YiIyiSxRUTrJLFFROsksUVE6ySxRUTrJLFFROsksUVE6/wfchBC52bJccYAAAAASUVORK5CYII=\n",
```

Comment:
Can we commit this without outputs? At least to get git to realize this is a rename and not a wholesale deletion and addition